apache/superset
# chore: Updates the bug report template

### SUMMARY
This PR changes the bug report template to make it clearer which versions are officially supported and remove unnecessary sections to simplify the reporting process. Specifically:
- It converts the issue template to an issue form
- It changes the Environment section from free form input to a set of dropdowns with predefined options
- It adds instructions to upgrade to the officially supported environment versions when necessary
- It removes the Expected/Actual sections as these are generally present in the bug description or can be easily deduced from it.
- It adds instructions to update the bug template when releasing new versions.

### TESTING INSTRUCTIONS
Check [GitHub's syntax for issue forms](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-issue-forms).

### ADDITIONAL INFORMATION
